.k8s_ci.Dockerfile

```diff
@@ -128,17 +122,14 @@ ARG GALAXY_USER
 ENV LC_ALL=en_US.UTF-8
 ENV LANG=en_US.UTF-8
 
-# Install python-virtualenv
 RUN set -xe; \
-    echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup \
-    && echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache \
+    echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache \
     && apt-get -qq update && apt-get install -y --no-install-recommends \
         locales \
-        libpython3.6 \
-        python3-virtualenv \
         vim-tiny \
         nano-tiny \
         curl \
+    && echo 'en_US.UTF-8 UTF-8' > /etc/locale.gen \
```

Comment:
@rhpvorderman What does this line do? Does it affect the ability to override the LANG with $LANG?

Reply:
In the Debian image the locale will not be generated unless it is in `/etc/locale.gen`. This is a difference with the ubuntu image. This line enables the en_US locale.

Reply:
Ah I see what you mean. I made it more generic so now $LANG is the one enabled in locale.gen.